lib/wor-prof.rb

```diff
@@ -0,0 +1,16 @@
+require 'wprof/version'
+require 'wprof/conf_wprof'
+require 'wprof/reporters/db_report'
+require 'wprof/reporters/external_report'
+require 'wprof/reporters/file_report'
+require 'wprof/wprof_reporter'
+require 'wprof/gen_prof'
+require 'wprof/wprof_subscriptors'
+
+module Wprof
+  def self.included(base)
+    load 'wprof/wprof_modules.rb'
+    base.extend(ForHttparty) if base.include?(HTTParty)
+    base.prepend(CustomMethods) unless WProf::Config.get_value(:custom_methods).nil?
```

Comment:
Thought this is okay I prefer `if` instead of `unless` so it is a little bit easier to read and we also follow the convention of this method:
```
base.prepend(CustomMethods) if WProf::Config.get_value(:custom_methods).present?
```

Reply:
Ok, is valid and i'm agreed, specially with the part "is a little bit easier to read". Done!